lib/workers/repository/update/pr/changelog/releases.ts

```diff
@@ -41,22 +41,41 @@ export async function getInRangeReleases(
     const pkgReleases = (await getPkgReleases(config))!.releases;
     const version = get(versioning);
 
-    const releases = pkgReleases
+    const previousReleases = pkgReleases
       .filter((release) =>
         version.isCompatible(release.version, currentVersion)
       )
-      .filter(
-        (release) =>
-          version.equals(release.version, currentVersion) ||
-          version.isGreaterThan(release.version, currentVersion)
-      )
       .filter((release) => !version.isGreaterThan(release.version, newVersion))
       .filter(
         (release) =>
           version.isStable(release.version) ||
           matchesUnstable(version, currentVersion, release.version) ||
           matchesUnstable(version, newVersion, release.version)
       );
+
+    const releases = previousReleases.filter(
+      (release) =>
+        version.equals(release.version, currentVersion) ||
+        version.isGreaterThan(release.version, currentVersion)
+    );
+
+    // if there is only 1 release it can be one of two things
+    // either there really is only 1 release OR
+    // the pinned version actually doesn't exist i.e pinning to 1.2.3 but only 1.2.4, 1.2.2 exists
```

Comment:
```suggestion
    /**
     * If there is only one release, it can be one of two things:
     *
     *   1. There really is only one release
     *
     *   2. Pinned version doesn't actually exist, i.e pinning `^1.2.3` to `1.2.3`
     *      while only `1.2.2` and `1.2.4` exist.
     */
```